Sources/VenmoPayments/VenmoClient.swift

```diff
@@ -1,6 +1,38 @@
 import Foundation
+#if canImport(CorePayments)
+import CorePayments
+#endif
 
 public class VenmoClient {
     
-    // TODO: to be implemented in a future PR
+    // MARK: - Properties
+    
+    private let config: CoreConfig
+    private let universalLink: URL
+    private let networkingClient: NetworkingClient
+    
+    // MARK: - Initializers
+    
+    ///  Initialize a VenmoClient to process a Venmo payment
+    ///  - Parameter config: The CoreConfig object
+    init(config: CoreConfig, universalLink: URL) {
+        self.config = config
+        self.universalLink = universalLink
+        self.networkingClient = NetworkingClient(coreConfig: config)
+    }
```

Comment:
```suggestion
    ///  Initialize a VenmoClient to process a Venmo payment
    ///  - Parameter config: The CoreConfig object
    convenience init(config: CoreConfig, universalLink: URL) {
        self.init(config: config, universalLink: universalLink, networkingClient: NetworkingClient(coreConfig: config))
    }
```
Take or leave: I noticed that the other constructor is primarily for testing. I'd add this suggestion only if it makes sense to DRY up the initialization logic.